xbmc/music/MusicDatabase.cpp

```diff
@@ -12128,7 +12128,7 @@ bool CMusicDatabase::ExportSongHistory(TiXmlNode* pNode, CGUIDialogProgress* pro
       XMLUtils::SetInt(song, "timesplayed", m_pDS->fv("iTimesplayed").get_asInt());
       XMLUtils::SetString(song, "lastplayed", m_pDS->fv("lastplayed").get_asString());
       auto* rating = XMLUtils::SetString(
-          song, "rating", StringUtils::FormatNumber(m_pDS->fv("rating").get_asFloat()));
+          song, "rating", fmt::format("{}",m_pDS->fv("rating").get_asFloat()).c_str());
```

Comment:
```suggestion
          song, "rating", fmt::format("{:.1f}",m_pDS->fv("rating").get_asFloat()));
```

Reply:
Indeed rating wants to be a 1 decimal place number, since it is always 10 or less the locale sensitive separators provided by  `FormatNumber` are irelevent.